pages.pl/windows/choco-upgrade.md

```diff
@@ -0,0 +1,32 @@
+# choco upgrade
+
+> Aktualizacja jednej lub więcej paczek zarządzanych przez Chocolatey.
+> Więcej informacji: <https://chocolatey.org/docs/commands-upgrade>.
+
+- Aktualizacja jednego lub więcej pakietów (oddzielonych spacją):
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
+
+- Aktualizacja pakietu do konkretnej wersji:
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
+
+- Aktualizacja wszystkich pakietów zarządzanych przez Chocolatey:
+
+`choco upgrade all`
+
+- Aktualizacja wszystkich pakietów zarządzanych przez Chocolatey z wyjątkiem podanych w parametrze --except:
```

Comment:
Rephrase this line to better align with the main page. (Upgrade all except specified comma-separated packages:)